src/view/view.js

```diff
@@ -200,14 +213,48 @@ export default class View {
 		// Set view root name the same as DOM root tag name.
 		viewRoot._name = domRoot.tagName.toLowerCase();
 
+		const initialDomRootAttributes = {};
```

Comment:
I think it should be `WeakMap` `root => Object ( attributeName => attributeValue )` The way it now means that you put attributes for all roots together and when you restore them they can be mixed. It will work fine only for a single root.

Reply:
It is like you say. Check out https://github.com/ckeditor/ckeditor5-engine/pull/1644/files#diff-0dfbdba2b2bbf34988f67a22d584575dR239.

Reply:
Ach, you're right. My bad.